haystack/components/audio/whisper_local.py

```diff
@@ -113,7 +113,7 @@ def run(self, sources: List[Union[str, Path, ByteStream]], whisper_params: Optio
                 alignment data and the path to the audio file used for the transcription.
         """
         if self._model is None:
-            raise ComponentError("The component was not warmed up. Run 'warm_up()' before calling 'run()'.")
+            raise RuntimeError("The component was not warmed up. Run 'warm_up()' before calling 'run()'.")
```

Comment:
I went with the "least amount of changes" strategy here, the majority of the components raise `RuntimeError`.